app/config/config.go

```diff
@@ -0,0 +1,42 @@
+// Package config works with config file for building pages
+package config
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+)
+
+// Domain struct is data about domain what we neet to evaluate
+type Domain struct {
+	Name        string `json:"name"`
+	PricePerDay int    `json:"price_per_day,omitempty"`
+	Rate        int    `json:"rate,omitempty"`
+}
+
+// Config struct is data about domais
+// and common values for domains
+// without overrided PricePerDay and Rate
+type Config struct {
+	Domains     []Domain `json:"domains"`
+	PricePerDay int      `json:"price_per_day"`
+	Rate        int      `json:"rate"`
+}
+
+// ReadConfig returns content of file
+// which was readed by provided filepath
+func ReadConfig(filepath string) Config {
+	fmt.Printf("Read config file: %q\n", filepath)
+	configBytes, err := ioutil.ReadFile(filepath)
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
```

Comment:
It's better to avoid `os.Exit` calls outside of `main` function: a usual pattern is to return errors to the caller of the function, and the caller (`main` in such case) will decide if it should retry, continue or halt the program.